README.md

```diff
@@ -1 +1,153 @@
-# Microverse-capstone-one
\ No newline at end of file
+# Microverse-capstone-one<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [Microverse-capstone-one](#microverse-capstone-one)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 First Capstone Project ](#-first-capstone-project-)
+  - [Project Description\*\*](#project-description)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://paulinelali.github.io/Microverse-capstone-one/index.html)
```

Comment:
- [ ] `Required`
- [ ] Good job implementing the requested changes!! 👏 
- [ ] However, your deployed site has some errors the logo is not appearing in the deployed project
- [ ] Please see screenshot below

![image](https://github.com/Paulinelali/Microverse-capstone-one/assets/66230306/6da5d3c4-7e4b-496f-a183-62105c5b823a)